components/CopyButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import { Button } from './theme/Button'
+import { AlertIcon, CheckIcon, CopyIcon } from '@primer/octicons-react'
+import * as Sentry from '@sentry/react'
+import { ColorTypes } from './theme/colors'
+
+enum State {
+  NotCopied,
+  Copied,
+  Error
+}
+
+enum ColorType {
+  Primary = 'primary',
+  Info = 'info',
+  Success = 'success'
+}
+
+const CopyButton: React.FC<{ value: any }> = ({ value }: { value: any }) => {
```

Comment:
We shouldn't use `any` as a typescript definition whenever possible, because using `any` as a type pretty much defeats the purpose of typescript

Reply:
Oh shoot. It was `clipboard.write` which accepts `any` value. Will change it to string.